ra2ce/graph/networks.py

```diff
@@ -552,7 +636,10 @@ def create(self) -> dict:
             # Create the network from the network source
             if self.config["network"]["source"] == "shapefile":
                 logging.info("Start creating a network from the submitted shapefile.")
-                base_graph, network_gdf = self.network_shp()
+                if self.cleanup:
+                    base_graph, network_gdf = self.network_shp()
+                else:
+                    base_graph, network_gdf = self.network_cleanshp()
 
```

Comment:
This is missleading, should not be the other way around? 
If cleanup -> self.network_cleanshp()...

```suggestion
                if self.cleanup:
                    base_graph, network_gdf = self.network_cleanshp()
                else:
                    base_graph, network_gdf = self.network_shp()

```

Reply:
Actually no... but I see where the confusion is and will make them clear in the `NetworkShpWrapper`